friture/pitch_tracker.py

```diff
@@ -102,17 +107,37 @@ def __init__(self, parent, engine):
 
         self.gridLayout.addWidget(self.quickWidget, 0, 0, 1, 1)
 
+        self.pitch_view_model = PitchViewModel(self)
+        pitch_window = QQuickWindow()
+        pitch_component = QQmlComponent(engine, qml_url("PitchView.qml"), self)
```

Comment:
Instead of having 2 views next to each other in the layout, could we have a new single view (that replaces Scope.qml) ?

My idea behind that is that at some point in the future, I would like to migrate the whole layout of Friture from QWidget to QML. So if each audio widget is a single QML component, that makes it easier.

Reply:
Yeah I had the same thought as I was putting together these pull requests. Pretty sure it would have saved a lot of hair pulling trying to get the widgets to size correctly if I'd thought of it to begin with :upside_down_face:. I think the other comment on the model probably dovetails nicely. 

I think I'd rather do that all as a follow-up, rather than amending this PR.

Reply:
Alright, let's do it in a follow-up PR then. I'm merging as is.